README.md

```diff
@@ -20,11 +20,83 @@ This terraform module creates an EC2 Instance with a security group, Cloudwatch
 - Follows aws security best practices and uses checkov to ensure compliance.
 - Has elaborate examples that you can use to setup your ec2 instance within a very short time.
 - Ability to create associated ec2 resources with minimum configuration changes.
+- This module includes a feature to install ssm agent and gives the necessary permissions for the instance to communicate with SSM manager.
+- As of [2.0.0] we no longer support or enable SSH keys on the instances, this is aligned with AWS best practices. As an alternative you should use Session Mananger which providers support to login to the Ec2 Linux and Windows (read below for instructions)
+
```

Comment:
Add:
`- This module as full support for Linux instances and partial support for Windows Instances (see changelog for Windows features unreleased).`